index.js

```diff
@@ -63,9 +63,24 @@ function normalizeVersions(versions) {
   }
 }
 
+const getChecksum = async (registryUrl, pkgName, version) => {
+  const {stdout} = await execa(`npm view --registry=${registryUrl} --@wix:registry=${registryUrl} --cache-min=0 ${pkgName}@${version} checksum`, {shell: true});
+  return stdout;
+};
+
+async function isSameHashAsPublished(pkgName, localVersion, currentPublishedVersion, registryUrl) {
```

Comment:
that's great, thank you!

I think this logic should be in the compare functions (`compare`, `compareVersion`) since we want to support checking with checksum both when comparing to published versions (in case of skip) and when comparing the local package (which you can assume a `checksum` field will be in the `package.json`.
We also need to add this field to the list of ignored fields cause the plan is to add this field to all published packages.

Reply:
sure. will do